lib/pbench/test/unit/server/conftest.py

```diff
@@ -158,6 +162,24 @@ def make_logger(server_config):
     return get_pbench_logger("TEST", server_config)
 
 
+@pytest.fixture()
+def capinternal(caplog):
+    def compare(message: str, response: Optional[Response]):
+        pat = re.compile(
+            f"INTERNAL error {message!r}: report error code [a-zA-Z\\d]{{8}}-([a-zA-Z\\d]{{4}}-){{3}}[a-zA-Z\\d]{{12}} to an SRE"
+        )
+        if response:
+            assert response.status_code == HTTPStatus.INTERNAL_SERVER_ERROR
+            assert pat.match(response.json["message"]), response.json["message"]
```

Comment:
I think that PyTest will report the arguments to the assertion if it fails (I think it will even report the arguments to `pat.match()`), so the "assertion message" here is somewhere between inadequate and redundant (i.e., it doesn't really describe what went wrong, it just reports what the message was...which is _not_ why the **test** failed).

On the other hand...at line 178 it _would_ be handy to know what the expected message was that we didn't find....

Reply:
Dumping a `Pattern` shows the string; I'm not sure what pytest will display for a failing `Pattern.match`, though, and this seemed easy enough.